model/comments.model.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection");
+
+exports.addCommentByArticleId = (article_id, commentToPost) => {
+  const commentReqBody = Object.keys(commentToPost);
+  const { username, body } = commentToPost;
+
+  if (
+    commentReqBody.length < 2 ||
+    commentReqBody[0] !== "username" ||
+    typeof commentReqBody[0] !== "string"
+  ) {
```

Comment:
I feel like it would be more efficient to check 'username' and 'body' are defined properties on the comment req body rather than specifically saying the 'username' key is in the keys arr you've made on line 4. You should be checking username AND body are present

Checking the number of keys is at least 2 is not a great check here, it could be covered by just checking `username` and `body` are present properties